src/uct/ib/base/ib_verbs.h

```diff
@@ -297,6 +297,24 @@ static inline void uct_ib_destroy_srq(struct ibv_srq *srq)
     }
 }
 
+static inline ucs_status_t uct_ib_qp_max_send_sge(struct ibv_qp *qp,
```

Comment:
why does it need to be inline?

Reply:
afaiu, we do not have proper `.c` file, so many funcs in this header would not need to be inline

Reply:
maybe ib_verbs.c is worth introducing, but probably in the another PR

Reply:
maybe move this to ud? since it's the only user now

Reply:
i use it in rc tests as well

Reply:
ok